src/MenuActionBar.cpp

```diff
@@ -605,7 +606,7 @@ void MenuActionBar::checkAction(std::vector<ActionData> &action_queue) {
 		}
 
 		// joystick/keyboard action button
-		else if (!inpt->usingMouse() && slots[i]->checkClick() == WidgetSlot::ACTIVATE) {
+		else if (!have_analog_aim && slots[i]->checkClick() == WidgetSlot::ACTIVATE) {
```

Comment:
I don't think this is correct, as it is handling the player clicking on an action bar slot as opposed to clicking on the play field.